packages/core/src/card/InteractableCard.tsx

```diff
@@ -31,7 +31,7 @@ export const InteractableCard = forwardRef<
   InteractableCardProps
 >(function InteractableCard(props, ref) {
   const {
-    accentPlacement = "bottom",
+    accentPlacement,
```

Comment:
We should decide if this is a breaking change. Arguably it is. But will be up to design to decide.